src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.star__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--0 > * {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 > *:nth-child(n+2) {
+  background-image: url("images/star.svg");
+}
+
+.stars--2 > *:nth-child(n+3) {
+  background-image: url("images/star.svg");
+}
+
+.stars--3 > *:nth-child(n+4) {
+  background-image: url("images/star.svg");
+}
+
+.stars--4 > *:nth-child(n+5) {
+  background-image: url("images/star.svg");
+}
```

```suggestion
.stars--0 > * ,
.stars--1 > *:nth-child(n+2),
.stars--2 > *:nth-child(n+3),
.stars--3 > *:nth-child(n+4),
.stars--4 > *:nth-child(n+5) {
  background-image: url("images/star.svg");
}
```

скомбінуй ці селектори